src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -656,6 +659,28 @@ else if (saturation > AVERAGE_SATURATION)
         }
     }
 
+    /**
+     * repair random equipped/held item with mending enchant.
+     *
+     * @param xp amount of xp available to mend with
+     * @return xp left after mending
+     */
+    private double applyMending(final double xp)
+    {
+        double localXp = xp;
```

Comment:
Quick question, what's the point in doing that? Declaring a value then immediately reassigning it? Please do correct me if I'm wrong =D